src/include/planner/alter_plan.h

```diff
@@ -0,0 +1,131 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// alter_plan.h
+//
+// Identification: src/include/parser/alter_plan.h
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+#include "planner/abstract_plan.h"
+#include "parser/alter_statement.h"
+namespace peloton {
+
+namespace parser {
+class AlterTableStatement;
+}
+namespace catalog {
+class Schema;
+}
+namespace storage {
+class DataTable;
+}
+
+namespace planner {
+/** @brief The plan used for altering
+ *  TODO: adding support for add/drop column
+ */
+class AlterPlan : public AbstractPlan {
+ public:
+  AlterPlan() = delete;
+
+  explicit AlterPlan(
+      const std::string &database_name, const std::string &table_name,
+      const std::vector<std::string> &dropped_columns,
+      const std::unique_ptr<catalog::Schema> &added_columns,
+      const std::unique_ptr<catalog::Schema> &changed_type_columns,
+      AlterType a_type);
+  explicit AlterPlan(const std::string &database_name,
```

Comment:
I believe you guys have changed rename columns from using vector to string, why still vector here?